src/query/service/src/sql/executor/format.rs

```diff
@@ -62,6 +63,9 @@ fn table_scan_to_format_tree(
     plan: &TableScan,
     metadata: &MetadataRef,
 ) -> Result<FormatTreeNode<String>> {
+    if plan.table_index == DUMMY_TABLE_INDEX {
```

Comment:
why specially handle it?

Reply:
For convenient, we can add a new `PhysicalPlan` for `DummyTableScan` instead of this.